### PR TITLE
[fix][websocket] Fix webSocketPingDurationSeconds config

### PR DESCRIPTION
**Pull request by nodece**
### Motivation

#19203 introduces `webSocketPingDurationSeconds` config, but doesn't add this config to the proxy and broker config class.

When the proxy/broker calls the `WebSocketService(ClusterData localCluster, ServiceConfiguration config)`, the ping feature will be broken.
 
### Modifications

- Add `webSocketPingDurationSeconds` to `ProxyConfiguration`
- Add `webSocketPingDurationSeconds` to `ServiceConfiguration`
- Improve reading the `webSocketPingDurationSeconds`

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository:

### CODE REVIEW

**Comment by nodece** [comment]
Ping @poorbarcode

File: pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java [hunk 1]
```diff
@@ -803,6 +803,12 @@ public class ProxyConfiguration implements PulsarConfiguration {
     )
     private boolean webSocketServiceEnabled = false;
 
+    @FieldContext(
+            category = CATEGORY_WEBSOCKET,
+            doc = "Interval of time to sending the ping to keep alive in WebSocket proxy. "
+                    + "This value greater than 0 means enabled")
+    private int webSocketPingDurationSeconds = -1;
```
[the remark this answers is not in the document]
Comment (nodece) [reply pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java:810]: We already add this to other branches, so I don't suggest rename to `keepAliveIntervalSeconds`.

File: pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java [hunk 2]
```diff
@@ -192,18 +191,15 @@ private void closePingFuture() {
     @Override
     public void onWebSocketConnect(Session session) {
         super.onWebSocketConnect(session);
-        WebSocketProxyConfiguration webSocketProxyConfig = service.getWebSocketProxyConfig();
-        if (webSocketProxyConfig != null) {
-            int webSocketPingDurationSeconds = webSocketProxyConfig.getWebSocketPingDurationSeconds();
```
Comment (poorbarcode) [inline pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java:197]: Does the field `webSocketPingDurationSeconds` in class `WebSocketProxyConfiguration` not useful now? If yes, should we delete it?

**Review by poorbarcode — COMMENTED** [review]
Should we also should edit the test added in https://github.com/apache/pulsar/pull/19203?

File: pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java (hunk 2 above)
Comment (nodece) [reply pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java:197]: No, these configurations can be converted to each other, so add `webSocketPingDurationSeconds` to each config.

**Comment by nodece** [comment]
branch-2.10 and branch-2.11 include these changes.

File: pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java (hunk 2 above)
Comment (poorbarcode) [reply pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java:197]: I see.

> Should we also should edit the test added in https://github.com/apache/pulsar/pull/19203?

And this comment ^_^

Comment (nodece) [reply pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java:197]: No. My changes don't break the tests of #19203.

File: pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Technoboy-) [reply pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java:810]: ok

**Review by Technoboy- — APPROVED** [review]

**Review by poorbarcode — APPROVED** [review]

**Pull request merged**